msu/systems/keybinds/keybind_sq.nut

```diff
@@ -2,14 +2,16 @@
 {
 	Function = null;
 	State = null;
+	KeyState = null;
 
 	constructor( _modID, _id, _keyCombinations, _state, _function, _name = null, _keyState = null)
```

Comment:
Seems a little awkward to me to have `keybind` that doesn't take in a `_keyState` arg but `keybind_sq` that does, in that you end up with statements like `msu_connection.js:39` where it calls `addKeybindFromSQ`. . .does "from SQ" imply that it's a keybind_sq, or a regular keybind?

Admittedly, very new to this code so it might be more obvious to people working with it more.